### ISSUE TITLE
Please let location also be set manually

### ISSUE THREAD

**Issue by noyannus** [skhzhang/time-based-themes#51]
**Is your feature request related to a problem? Please describe.**

No problem.

**Describe the solution you'd like**

An additional option on the settings page to enter longitude/latitude for the sunrise/sunset calculation.

Useful whenever location services are broken or unavailable.

**Comment by Medullitus** [comment]
+++++++++

**Comment by elhennig** [comment]
Yes, please let the users enter the Lon Lat coordinates manually
